app/assets/javascripts/components/revisions/diff_viewer.jsx

```diff
@@ -19,7 +19,8 @@ const DiffViewer = React.createClass({
 
   getInitialState() {
     return {
-      showDiff: false
+      showDiff: false,
+      chars: this.props.charCount
```

Comment:
This character count is not being used yet? In any case, if you're pulling it from props, then there's no need to copy it to state. You can just do `this.props.revision.characters` whenever you want to access that value.